### ISSUE TITLE
Select teams as reviewers

### ISSUE THREAD

**Issue by pagameba** [github/VisualStudio#2590]
**Is your feature request related to a problem? Please describe.**
We use GitHub teams to assign reviewers to pull requests, GitHub can be configured to replace the team with a random set of users from the team to ensure that pull request reviews get evenly distributed to the team members.  When assigning reviewers using this extension to VS Code, we can only select individual people.

**Describe the solution you'd like**
We would like to be able to select Teams to be assigned as reviewers.

**Describe alternatives you've considered**
We currently either manually assign reviewers or click the link to the PR to assign the team.

**Comment by lsanwick** [comment]
Yeah, this would be really great to have, as my team heavily leverages the "team" assignment for code reviews.
